res/css/views/right_panel/_UserInfo.scss

```diff
@@ -52,6 +52,19 @@ limitations under the License.
 
     .mx_UserInfo_container {
         padding: 8px 16px;
+
```

Comment:
there appears to be a lot of refactoring in this file which makes it hard to review - please remove any refactoring and instead include only the necessary changes.

Reply:
If you have not noticed yet, you could check those refactoring per commits, divided into parts which should be as simple as possible, but if you don't have much time to follow them and verify, I would cherry-pick some commits to achieve an expected design. Nevertheless, I would create another PR anyway to refactor the current style rules- there is a complex dependency which nobody should be able to comprehend at first, such as `a .mx_UserInfo_avatar > div > div *`.

Reply:
Please wait for a moment- I need some time to restructure the PR.